laravel/app/ScheduleParser.php

```diff
@@ -46,13 +43,20 @@ private static function getLineDate($line)
      */
     private static function getLineTime($line, $index)
     {
+        $time = "00:00:00";
+        $length = strlen($line[$index]); 
         if (strcmp($line[$index], "") == 0) {
-            return null;
+            $time = NULL;
```

Comment:
I'm a little confused by this. Do you mind explaining what the condition is for? Perhaps even with a comment in the code.

Reply:
It means the cell is empty - meaning no specified time => null

Reply:
Understood. In the interest of being consistent with the other conditions/branches, do you think it makes more sense to do `if ($length == 0) {`?

Other than that, I'm good with this PR, so I'll approve now and I trust your judgement either way!